Questions.ViewModelComposition/QuestionsModule.cs

```diff
@@ -0,0 +1,75 @@
+﻿using ITOps.Json;
+using ITOps.ViewModelComposition.Modules;
+using Questions.ViewModelComposition.Events;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Net.Http;
+
+namespace Questions.ViewModelComposition
+{
+    public class QuestionsModule : RequestsModule
+    {
+        public QuestionsModule()
+        {
+            Get("/questions", async (requestId, vm) =>
+            {
+                var url = $"http://localhost:54217/api/questions";
+                var client = new HttpClient();
+                var response = await client.GetAsync(url);
+
+                dynamic[] questions = await response.Content.AsExpandoArray();
+
+                if (questions.Length == 0)
+                {
+                    vm.Questions = questions;
+                    return;
+                }
+
+                IDictionary<dynamic, dynamic> questionsViewModel = MapToDictionary(questions);
+
+                await vm.RaiseEvent(new QuestionsLoaded()
```

Comment:
Unrelated to this PR but it's not obvious that the event handlers are executed synchronously here and that they are all successfully completed when you do `vm.Questions = questionsViewModel.values.ToList()`. Some options:
- developers just need to learn it
- the RaiseEvent method could be renamed to be more explicit, like RaiseEventAndExecuteSubscribers (ugly, I know)
- explicitly ask for a continuation `await vm.RaiseEvent(new ... , () => { vm.Result = "It worked";}, ()=> { vm.Result = "It failed";});

Reply:
documentation is probably a much better approach